quickwit-config/src/index_config.rs

```diff
@@ -306,17 +311,21 @@ pub fn build_doc_mapper(
     search_settings: &SearchSettings,
     indexing_settings: &IndexingSettings,
 ) -> anyhow::Result<Arc<dyn DocMapper>> {
-    let mut builder = DefaultDocMapperBuilder::new();
```

Comment:
That change may seem weird... 

The way we build DefaultDocMapper is actually quite unexpected.
We both build it through deserializing and through a `DocMapping` object.

For this reason it is quite easy to forget modify both location if we add a field for instance.

Relying on struct initialization makes this kind of mistakes  harder.

Reply:
Thanks for cleaning that, I think it comes from the 0.2 hurry release.